test/cli/migrate-make.spec.js

```diff
@@ -1,11 +1,57 @@
 'use strict';
 
+const fs = require('fs');
 const path = require('path');
 const { FileTestHelper, execCommand } = require('cli-testlab');
 const expect = require('chai').expect;
 
 const KNEX = path.normalize(__dirname + '/../../bin/cli.js');
 
+// start: utils for --stub option tests //
+const migrationStubSetup = (fileHelper) => {
```

Comment:
Could you please use `function migrationStubSetup (fileHelper) {` style instead?